### PR TITLE
Sorting visualizer

### LINKED ISSUE #110: Sorting Visualizer
**Describe the project you want to add with tech stack**

This project is a Sorting visualizer made using HTML, CSS, and JS which helps visualize Bubble Sort Algorithm.
**Expected behavior**

when the user clicks the generate button an array with random number is generated. 
when the user clicks the sort button the array is sorted.

**Screenshots (optional)**

Screenshot 2023-10-06 205849

**Additional context (optional )**

### PR DESCRIPTION
**Pull request by viditagrawal56**
# Title and Issue number 

Title : Sorting visualizer

Issue No. :#110

Code Stack : HTML, CSS and JS

Close #110 

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`
- [X] I have updated the Index.html file for my contribution

### CODE REVIEW

File: native/machine-coding/sorting-visualizer/index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet" />
```
Comment (sadanandpai) [inline native/machine-coding/sorting-visualizer/index.html:10]: The font is not needed. we have a standard font across

**Comment by sadanandpai** [comment]
@viditagrawal56 Nice project.

we have a separate project for sorting. https://github.com/sadanandpai/sorting-visualizer

File: native/machine-coding/sorting-visualizer/index.html (hunk 1 above)
Comment (viditagrawal56) [reply native/machine-coding/sorting-visualizer/index.html:10]: Oh sorry I will fix this

Comment (viditagrawal56) [reply native/machine-coding/sorting-visualizer/index.html]: @sadanandpai Fixed :)

**Comment by viditagrawal56** [comment]
@sadanandpai Is this PR ready to merge or do you want me to make some changes?

**Pull request merged**